app/views/groups/_form.html.erb

```diff
@@ -0,0 +1,24 @@
+<% if group.errors.any? %>
+  <div class="post-errors-box">
+    <p><%= pluralize(@group.errors.count, "error") %> found:</p>
+
+    <ul class="post-errors-ul">
+      <% group.errors.full_messages.each do |message| %>
+        <li><%= message %></li>
+      <% end %>
+    </ul>
+  </div>
+<% end %>
+<%= form_with model: group, class: 'flex flex-col w-full max-w-3xl mx-auto gap-6 justify-center items-center' do |f| %>
+  <div class= 'w-full max-w-lg mx-auto mt-6 mb-4'>
+    <div class="w-full max-w-lg mx-auto">
+      <%= f.text_field :name, autofocus: true, autocomplete: "text", placeholder: "Name", class: "w-full" %>
+    </div>
+    <div class="w-full max-w-lg mx-auto">
+      <%= f.text_field :icon, autofocus: true, autocomplete: "text", placeholder: "Icon URL", class: "w-full" %>
```

Comment:
- [ ] Well done implementing all the required changes however the category icon image still too big. Kindly adress the issue as required by the previous TSE.


<img width="442" alt="Screenshot 2023-09-15 at 9 55 06 PM" src="https://github.com/lordksix/budget-app-by-rails/assets/53347966/7f6c5bc7-9b87-42fc-be95-de3dca332b58">

Reply:
Thank you for the feedback. All remaining images has been fixed.